### PR TITLE
Remove selected artifacts

### PR DESCRIPTION
**Pull request by MDrakos**
https://www.pivotaltracker.com/story/show/173526871

### CODE REVIEW

File: internal/runners/clean/cache.go [hunk 1]
```diff
@@ -44,6 +55,27 @@ func (c *Cache) Run(params *CacheParams) error {
...
+		ok, fail := c.confirm.Confirm(locale.Tr("clean_cache_artifact_confirm", namespace), false)
+		if fail != nil {
+			return fail.ToError()
+		}
+		if !ok {
+			return nil
+		}
+	}
+
+	split := strings.Split(namespace, "/")
```
Comment (Naatan) [inline internal/runners/clean/cache.go:73]: http://github.com/ActiveState/cli/blob/c06b9f831c3b6bddb2a27fd50e9e4539df3895d1/pkg/project/namespace.go#L71-L71

File: internal/runners/clean/cache.go [hunk 2]
```diff
@@ -44,6 +55,27 @@ func (c *Cache) Run(params *CacheParams) error {
...
+	if len(split) != 2 {
+		return locale.NewInputError("err_clean_cache_invalid_namespace", "Namespace argument is not in the correct format of <Owner>/<ProjectName>")
+	}
+	projectInstallPath := runtime.InstallPath(split[0], split[1])
+
+	logging.Debug("Remove project path: %s", projectInstallPath)
+	return os.RemoveAll(projectInstallPath)
```
Comment (Naatan) [inline internal/runners/clean/cache.go:80]: This error needs to be locale wrapped.

File: internal/runners/clean/cache.go (hunk 1 above)
Comment (MDrakos) [reply internal/runners/clean/cache.go]: I even looked for that function! 🤦

File: internal/runners/clean/cache.go [hunk 3]
```diff
@@ -70,12 +70,17 @@ func (c *Cache) removeProject(namespace string, force bool) error {
...
+		return locale.WrapError(fail.ToError(), "err_clean_cache_invalid_namespace", "Namespace argument is not of the correct format")
 	}
-	projectInstallPath := runtime.InstallPath(split[0], split[1])
+	projectInstallPath := runtime.InstallPath(parsed.Owner, parsed.Project)
 
 	logging.Debug("Remove project path: %s", projectInstallPath)
-	return os.RemoveAll(projectInstallPath)
+	err := os.RemoveAll(projectInstallPath)
+	if err != nil {
+		return locale.WrapError(err, "err_clean_remove_artifact", "Could not remove artifact for project: {{.V0}}", namespace)
```
Comment (Naatan) [inline internal/runners/clean/cache.go:82]:
```suggestion
		return locale.WrapError(err, "err_clean_remove_artifact", "Could not remove cached runtime environment for project: {{.V0}}", namespace)
```

**Pull request merged**